lib/express/call-context-middleware.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const uuid = require('uuid').v4;
+const CallContext = require('../call-context');
+
+module.exports = function (getConfig, getLogger, getServiceLocator, setContext) {
```

Comment:
Instead of passing the "get" functions, maybe it's possible to receive the objects themselves.